### PR TITLE
added db connection, cleaned up app.js file, reinitialized auth 

### PR DESCRIPTION
**Pull request by rs-217**
Here is the most recent auth handler without passport.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -2,108 +2,99 @@
...
+ */
 const express = require('express');
 const path = require('path');
-const passport = require('passport');
-const bodyParser = require('body-parser');
-const LocalStrategy = require('passport-local');
+const mysql = require('mysql');
 const bcrypt = require('bcrypt');
-const session = require('express-session');
+const bodyParser = require('body-parser');
```
Comment (BrianLinggadjaja) [inline app.js]: I think body-parser is deprecated on express.js
https://medium.com/@mmajdanski/express-body-parser-and-why-may-not-need-it-335803cd048c

File: app.js [hunk 2]
```diff
@@ -2,108 +2,99 @@
...
-    res.json(users); // return users array
+
+/**
+ * --------------DATABASE CONFIG-----------------
+ */
+let connection = mysql.createConnection({
+    host: "localhost",
+    user: "rs-217",
+    password: "$udoPowers37",
+    database:'node_app'
```
Comment (BrianLinggadjaja) [inline app.js]: We will eventually use .env variables after you're done testing and reference it using process.env.ENV_VAR

**Review by BrianLinggadjaja — APPROVED** [review]
Doing great :)

File: app.js (hunk 1 above)
Comment (rs-217) [reply app.js]: gotcha ill go ahead and fix that

File: app.js (hunk 2 above)
Comment (rs-217) [reply app.js]: Roger that!

**Comment by rs-217** [comment]
Connected MySQL database
- hid credentials in .env file

Database manipulation
- users who are created are stored in database
- check to see if user info is in database
- store user credentials in encrypted algo

**Comment by BrianLinggadjaja** [comment]
The .env creds are exposed

**Pull request merged**